lib/index.ejs

```diff
@@ -60,6 +63,13 @@
                             libraries: libs.map(lib => lib.default),
                         }
 
+                        const urlParams = new URLSearchParams(window.location.search);
+                        const locale = urlParams.get("locale");
```

Comment:
You might be able to just abbreviate to this? I haven't tested it but it should be equivalent:
```
options.locale = urlParams.get("locale");
```

Reply:
Good call. Done.